### PR TITLE
Gh 1265

### PR DESCRIPTION
**Pull request by FrankMittelbach**
# Internal housekeeping

## Status of pull request

- Ready to merge

## Checklist of required changes before merge will be approved
- [x] Test file(s) added
- [x] Version and date string updated in changed source files
- [x] Relevant `\changes` entries in source included
- [x] Relevant `changes.txt` updated
- [n/a] Rollback provided (if necessary)?
- [x] `ltnewsX.tex` (and/or `latexchanges.tex`) updated

### CODE REVIEW

**Review by davidcarlisle — APPROVED** [review]
looks good to me, I pushed one trivial comment typo fix to the branch

File: base/doc/ltnews40.tex [hunk 1]
```diff
@@ -302,7 +303,24 @@ \section{Avoid code duplication in rollback}
...
-%\section{Changes to packages in the \pkg{amsmath} category}
+
+
+\section{Changes to packages in the \pkg{amsmath} category}
+
+\section{Extend support for \cs{dots}}
+
+The implementation of \cs{dots} in \pkg{amsmath} has the feature that
+it selects different dots depending on the symbol that follows, e.g.,
+dots between, commas would be normally at the baseline, while dots
```
Comment (car222222) [inline base/doc/ltnews40.tex:314]: . . . on the baseline, ...

File: base/doc/ltnews40.tex [hunk 2]
```diff
@@ -302,7 +303,24 @@ \section{Avoid code duplication in rollback}
...
 
-%\section{Changes to packages in the \pkg{amsmath} category}
+
+
+\section{Changes to packages in the \pkg{amsmath} category}
+
+\section{Extend support for \cs{dots}}
+
+The implementation of \cs{dots} in \pkg{amsmath} has the feature that
+it selects different dots depending on the symbol that follows, e.g.,
```
Comment (car222222) [inline base/doc/ltnews40.tex:313]: . . . follows: . . .

File: base/doc/ltnews40.tex (hunk 1 above)
Comment (car222222) [inline base/doc/ltnews40.tex:314]: dots between commas . . . 
(no comma before commas:-)

File: required/amsmath/amsmath.dtx [hunk 3]
```diff
@@ -1139,15 +1147,37 @@ Foreign command \@backslashchar#1;\MessageBreak
...
-% In previous versions \verb|\long| macros were not seen by the lokkahead.
+%    In case \cs{\@let@token} is a robust \LaTeXe{} command, i.e.,
+%    expands to \cs{protect}
+%    \verb*=\somename =, we save the \cs{meaning} of
+%    \verb*=\somename = in \cs{meaning@} (possibly followed by some dots that have
+%    not been consumed as arguments during the expansion).
+%    Otherwise, we save the
+%    \cs{meaning} of \cs{@let@token} in \cs{meaning@}.  The dots at the end
+%    as well as the space are
+%    used in later parts of the checking (could probably be cleaned up a bit).
```
Comment (car222222) [inline required/amsmath/amsmath.dtx:1158]: Maybe:
. . . (this could . . .

File: required/amsmath/amsmath.dtx [hunk 4]
```diff
@@ -1139,15 +1147,37 @@ Foreign command \@backslashchar#1;\MessageBreak
...
+% \changes{v2.15d}{2016/06/28}{Add space token to prevent runaway
+%    argument error}
+% \changes{v2.17r}{2024/06/29}{Drop \cs{protect} if present (gh/1265)}
+%    \begin{macrocode}
+       \begingroup
+         \def\protect{\protect}% % make it a quark
```
Comment (car222222) [inline required/amsmath/amsmath.dtx:1164]: Hope this is safe!  
Is it clearly local? -- Yes, good!

File: required/amsmath/amsmath.dtx [hunk 5]
```diff
@@ -1208,6 +1238,15 @@ Foreign command \@backslashchar#1;\MessageBreak
  \thedots@}
 %    \end{macrocode}
 %
+%    The helper to generate the meaning of \cs{\@let@token} or its
```
Comment (car222222) [inline required/amsmath/amsmath.dtx:1241]: Additional comma and "of" needed here, to give:
. . . , or of its

File: required/amsmath/changes.txt [hunk 6]
```diff
@@ -2,6 +2,12 @@
 All changes above are only part of the development branch for the next release.
 ================================================================================
 
+2024-08-02  Frank Mittelbach  <Frank.Mittelbach@latex-project.org>
+
+	* amsmath.dtx (section{Ellipsis dots}):
+	Support special \dots magic in case of robust characters via \protected
```
Comment (car222222) [inline required/amsmath/changes.txt:8]: Extend the . . . to cover the case of ...

**Review by car222222 — COMMENTED** [review]
A very few of my suggestions are essential "wrong commas", and similar.  (Not that I am at all expert in the much-disputed area of English comma-usage:-)

File: required/amsmath/amsmath.dtx (hunk 4 above)
Comment (FrankMittelbach) [reply required/amsmath/amsmath.dtx:1164]: :-)

File: required/amsmath/changes.txt (hunk 6 above)
Comment (FrankMittelbach) [reply required/amsmath/changes.txt:8]: don't think so: it  supports 2 cases: the one using \protected\def and the one with the symbol being defined with \DeclareRobustcommand. Tried a different rewrite.

File: base/doc/ltnews40.tex [hunk 7]
```diff
@@ -302,7 +303,24 @@ \section{Avoid code duplication in rollback}
...
-%\section{Changes to packages in the \pkg{amsmath} category}
+
+
+\section{Changes to packages in the \pkg{amsmath} category}
+
+\section{Extend support for \cs{dots}}
+
+The implementation of \cs{dots} in \pkg{amsmath} has the feature that
+it selects different dots depending on the symbol that follows: e.g.,
+dots between commas would be normally on the baseline, while dots
```
Comment (car222222) [inline base/doc/ltnews40.tex:314]: Sorry I missed this one earlier:
. . . would normally be . . .

is more normal, as we might say, but this is not a very important change.

**Review by car222222 — APPROVED** [review]
One more spotted! Not vital.

**Pull request merged**